src/components/visit/Buttons.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { useState } from 'react'
+
+import Modal from './Modal'
+
+export default function EditButton(prop: { isEdit: boolean }) {
+  if (prop.isEdit) {
+    return <CancelButton />
+  } else {
```

Comment:
This could be a bit cleaner if you make use of early returns, see this article for more https://medium.com/swlh/return-early-pattern-3d18a41bba8